ocs_ci/ocs/bucket_utils.py

```diff
@@ -1912,3 +1964,142 @@ def create_aws_bs_using_cli(
         use_yes=True,
     )
+
+
+def get_object_creation_time_from_noobaa_db(bucket_name, object_key):
+    """
+    Get the creation time of a given object by querying the noobaa-db
+
+    """
+    psql_cmd = (
+        "SELECT data ->> 'create_time' "
+        "FROM objectmds "
+        "WHERE data ->> 'bucket' IN ( "
+        "SELECT _id "
+        "FROM buckets "
+        f"WHERE data ->> 'name' = '{bucket_name}') "
+        f"AND data ->> 'key' = '{object_key}' "
+    )
+
+    output = exec_nb_db_query(psql_cmd)[0].strip()
+    return output
+
+
+def change_objects_creation_date_in_noobaa_db(
+    bucket_name, object_keys, new_creation_time
+):
+    """
+    Change the creation date of given objects one year back at the noobaa-db
+
+    Args:
+        bucket_name (str): The name of the bucket where the objects reside
+        object_keys (list): A list of object keys to change their creation date
+        new_creation_time (int): The new creation time in unix timestamp in seconds
+
+    Example usage:
+        # Change the creation date of objects obj1 and obj2 in bucket my-bucket to one minute back
+        change_objects_creation_date("my-bucket", ["obj1", "obj2"], time.time() - 60)
+
+    """
+    if not object_keys:
+        return
+
+    psql_query = (
+        "UPDATE objectmds "
+        "SET data = jsonb_set(data, '{create_time}', "
+        f"to_jsonb(to_timestamp({new_creation_time}))) "
+        "WHERE data->>'bucket' IN ( "
+        "SELECT _id "
+        "FROM buckets "
+        f"WHERE data->>'name' = '{bucket_name}') "
+        f"AND data->>'key' = ANY(ARRAY{object_keys});"
+    )
+
+    exec_nb_db_query(psql_query)
+
+
+def expire_mcg_objects(bucket_name, object_keys, prefix=""):
+    """
+    Expire given objects in a bucket by changing their creation date to one year back
+
+    Note that this is a workaround for the fact that the shortest expiration
+    time that expiraiton policies allows is 1 day, which is too long for the tests to wait.
+
+    Args:
+        bucket_name (str): The name of the bucket where the objects reside
+        object_keys (list): A list of object keys to expire
+        prefix (str): The prefix of the objects to expire
+
+    """
+    logger.info(
+        f"Expiring objects in bucket {bucket_name} by changing their creation date"
+    )
+    object_keys = [prefix + key for key in object_keys]
+    SECONDS_IN_YEAR = 60 * 60 * 24 * 365
+    change_objects_creation_date_in_noobaa_db(
+        bucket_name, object_keys, time.time() - SECONDS_IN_YEAR
+    )
+
+
+def get_object_count_in_bucket(io_pod, bucket_name, prefix="", s3_obj=None):
+    """
+    Get the total number of objects in a bucket
+
+    Args:
+        io_pod (pod): The pod which should handle all needed IO operations
+        bucket_name (str): The name of the bucket to count the objects in
+        prefix (str): The prefix to start the count from
+        s3_obj (MCG or OBJ): An MCG or OBC class instance
+
```

Comment:
same as above, please add docstring for return

Reply:
Fixed 👍